app/models/katello/glue/pulp/repo.rb

```diff
@@ -519,7 +529,7 @@ def find_distributor(use_clone_distributor = false)
                          end
                        end
 
-        distributors.detect { |dist| dist["distributor_type_id"] == dist_type_id }
+        distributors&.detect { |dist| dist["distributor_type_id"] == dist_type_id }
```

Comment:
This allows calling `Katello::Repository.new.pulp_checksum_type` without errors.